frontend/packages/dev-console/src/components/pipelines/modals/triggers/AddTriggerForm.scss

```diff
@@ -0,0 +1,14 @@
+.odc-add-trigger-form {
+  &__variable-container {
+    border: 1px solid #ccc;
```

Comment:
Probably better to use a Patternfly var for border here like `--pf-global--BorderColor--100`. Not sure exactly what shade of grey you want here but they have quite a few to choose from.

Reply:
Oh rubbish... I forgot, this needs to get updated anyways. I hacked it in early on because I needed to talk to UX, and there was no component for it. Will update. Good catch.